src/Common/Core/Impl/Services/CoreServices.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.Common.Core.IO;
+using Microsoft.Common.Core.Logging;
+using Microsoft.Common.Core.OS;
+using Microsoft.Common.Core.Shell;
+using Microsoft.Common.Core.Telemetry;
+
+namespace Microsoft.Common.Core.Services {
+    [Export(typeof(ICoreServices))]
+    public sealed class CoreServices : ICoreServices {
+        private readonly IApplicationConstants _appConstants;
+        private readonly ILoggingServices _loggingServices;
+        private IActionLog _log;
+
+        [ImportingConstructor]
+        public CoreServices(
+              IApplicationConstants appConstants
+            , ITelemetryService telemetry
+            , ILoggingServices loggingServices
```

Comment:
Got it

---
In reply to: [81220028](https://github.com/Microsoft/RTVS/pull/2408#discussion_r81220028) [](ancestors = 81220028)